scm-ui/ui-webapp/src/repos/components/changesets/ContributorTable.tsx

```diff
@@ -69,11 +69,11 @@ const ContributorTable: FC<Props> = ({ changeset }) => {
     if (!changeset.contributors) {
       return [];
     }
-    return new Set(changeset.contributors.map((contributor) => contributor.type));
+    return Array.from(new Set(changeset.contributors.map(contributor => contributor.type)));
```

Comment:
Can we improve the code around this ? The `Set` seems to be used to ensure unique values, maybe we can refactor the whole block to still match the criteria but be overall better with typings ?